quarkus/service/src/test/java/org/apache/polaris/service/quarkus/catalog/GetConfigTest.java

```diff
@@ -39,7 +39,14 @@ public class GetConfigTest {
   @ValueSource(booleans = {true, false})
   public void testGetConfig(boolean enableGenericTable) {
     TestServices services =
-        TestServices.builder().config(Map.of("ENABLE_GENERIC_TABLES", enableGenericTable)).build();
+        TestServices.builder()
+            .config(
+                Map.of(
+                    "ENABLE_GENERIC_TABLES",
+                    enableGenericTable,
+                    "SUPPORTED_CATALOG_STORAGE_TYPES",
```

Comment:
For Quarkus test, I believe it should be possible to put this setting in a test profile in `application-test.properties` and/or `application-it.properties` and reduce the amount of code changes in tests.

Reply:
Good call, I'll try to remove this from all the Quarkus tests I can. This particular test seems to still need it even after updating `application-test.properties` though